Support/lib/ruby_utils.rb

```diff
@@ -0,0 +1,55 @@
+module RubyUtils
+  module_function
+
+  # Try to find the Ruby executable `name`. Return an array containing the
+  # command(s) needed to run the executable, or `nil` if the executable could
+  # not be found.
+  #
+  # Typical usage looks like this:
+  #
+  #     Dir.chdir(ENV['TM_PROJECT_DIRECTORY']) if ENV['TM_PROJECT_DIRECTORY']
+  #     executable = RubyUtils.find_executable('rspec')
+  #     if executable
+  #       system(*executable, '--some', '--additional', 'args')
+  #     else
+  #       # Executable not found, so use fallback / display alert / …
+  #     end
+  #
+  # Supports the following cases:
+  #
+  # 1. If an appropriate `TM_*` environment variable is present and points to an
```

Comment:
> But then I realized that the env var is the most “finegrained” setting

I agree, the environment variable is there for when the default behavior is undesured and require user intervention, so it should trump everything else.